.travis.yml

```diff
@@ -33,5 +34,7 @@ before_install:
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --dev --no-update; fi
 
 install:
-  - if [ "$deps" = "" ]; then composer install; fi
-  - if [ "$deps" = "low" ]; then composer update --prefer-lowest --prefer-stable --prefer-dist; fi
+  - composer update $COMPOSER_OPTIONS
+
+script:
+  - vendor/bin/phpunit
```

Comment:
We need this as the PHPUnit version that is shipped with Travis CI is built with `symfony/yaml` 3.1.x which may conflict with the installed version of the DependencyInjection component (the `Yaml::PARSE_CONSTANT` class constant is missing then).